doc/api/globals.md

```diff
@@ -531,6 +531,40 @@ changes:
 
 A browser-compatible implementation of the [`fetch()`][] function.
 
+```mjs
+const res = await fetch('https://nodejs.org/api/documentation.json');
+if (res.ok) {
+  const data = await res.json();
+  console.log(data);
+}
+```
+
+The implementation is based upon [undici](https://undici.nodejs.org), an HTTP/1.1 client written from scratch for Node.js. You can figure out which version of `undici` is bundled in your Node.js process reading the `process.versions.undici` property.
+
+## Custom dispatcher
+
+You can use a custom dispatcher to dispatch requests passing it in fetch's options object.
+The dispatcher must be compatible with `undici`'s [`Dispatcher` class](https://undici.nodejs.org/#/docs/api/Dispatcher.md).
+
+```js
+fetch(url, { dispatcher: new MyAgent() });
+```
+
+It is possible to change the global dispatcher in Node.js using:
+
+```js
+globalThis[Symbol.for('undici.globalDispatcher.1')] = yourDispatcher;
```

Comment:
@nodejs/undici I think we should provide an api in Node to override the global dispatched, its being used in the wild
https://github.com/search?q=globalThis%5BSymbol.for%28%27undici.globalDispatcher.1%27%29%5D&type=code

Reply:
I would not recommend to use the Symbol. Let's recommend to install undici and set.

@marco-ippolito we have a session for this at the collab summit in Paris.

Reply:
I have updated the doc replacing the Symbol with a setGlobalDispatcher() call